### PR TITLE
improve design issue + rating for outing on card/list

### PR DESCRIPTION
**Pull request by stef74**
Fix for issue #1808 and other improvement
and #1801 

- [x] Trash position
- [x] fix alignement of more activities
- [x] move author on same line than area (to offer more place for rating)
- [x] reduce size menu
- [x] remove author on post forum
- [x] show author name on hover on the avatar
- [x] remove time of the post
- [x] remove text for app mobile
- [x] densify (base on @cbeauchesne proposition)
- [x] add rating on feed, card, list with some adaptation 
- [x] show rating on outing from outing not from route

### CODE REVIEW

**Comment by stef74** [comment]
Som screenshoot with different resolution

capture d ecran 2017-11-26 a 19 20 45
capture d ecran 2017-11-26 a 19 19 30
capture d ecran 2017-11-26 a 19 19 13
capture d ecran 2017-11-26 a 19 17 53

**Comment by stef74** [comment]
@c2corg/developers anyone to review ?

**Comment by desnoes** [comment]
I found what seems to be an error in  c2corg_ui/templates/outing/helpers/view.html (line 295).
"<%def name="get_outing_rating(ouring)">" I guess ouring is an error and should be replace by outing.
I am not competent to review the remaining part of the code.

File: c2corg_ui/templates/outing/helpers/view.html [hunk 1]
```diff
@@ -292,33 +292,33 @@ <h3 ng-click="mainCtrl.animateHeaderIcon($event)" data-toggle="collapse" data-ta
 </%def>
 
 ## RATING
-<%def name="get_outing_route_rating(route)">\
-% if route.get('global_rating') or route.get('engagement_rating') or route.get('risk_rating') or route.get('exposition_rock_rating') \
-  or route.get('rock_free_rating') or route.get('rock_required_rating') or route.get('hiking_rating') \
-  or route.get('hiking_mtb_exposition') or route.get('ice_rating') or route.get('mixed_rating') or route.get('snowshoe_rating')  \
-  or route.get('mtb_down_rating') or route.get('mtb_up_rating') or route.get('via_ferrata_rating') or route.get('equipment_rating') \
-  or route.get('ski_rating') or route.get('labande_ski_rating') or route.get('labande_global_rating') or route.get('ski_exposition') :
+<%def name="get_outing_rating(ouring)">\
```
Comment (asaunier) [inline c2corg_ui/templates/outing/helpers/view.html]: As mentioned by @desnoes it should be "outing" instead of "ouring".

Comment (asaunier) [reply c2corg_ui/templates/outing/helpers/view.html]: Please note that all those attributes are not available in outing documents. The list of available attributes is here: https://github.com/c2corg/v6_api/issues/685#issuecomment-333570449

File: less/cards.less [hunk 2]
```diff
@@ -618,6 +621,21 @@
         .outing-date {
           width: 65px;
         }
+        .outing-icon-infos {
+          width:  150px;
+        }
+        
+        .area
+        {
```
Comment (asaunier) [inline less/cards.less]: Should be on the previous line

**Comment by asaunier** [comment]
@desnoes Good catch! To report an error or post a comment, it's more convenient to do it directly in the diff https://github.com/c2corg/v6_ui/pull/1829/files => click on the "+" that appears to the left when the mouse cursor hovers the code.

File: c2corg_ui/templates/index.html [hunk 3]
```diff
@@ -74,15 +73,14 @@ <h3 translate>Mobile application</h3>
...
+          <a href="${discourse_url}t/{{::post['slug']}}/{{::post['id']}}" title="{{::post['last_poster_username']}}">
             <div class="fpcontent">
               <div class="fleft">
                 <div class="fportrait">
                   <img ng-src="${discourse_url}{{::post['avatar_template']}}">
                 </div>
                 <div class="ftxt">
-                  <div class="fauthor">{{::post['last_poster_username']}}</div>
-                  <div class="ftitle"><span class="title">{{::post['title']}}</span> - {{::post['last_posted_at'] | amUtc | amLocal | amTimeAgo}}</div>
+                  <div class="ftitle"><span class="title">{{::post['title']}}</span></div>
```
Comment (asaunier) [inline c2corg_ui/templates/index.html]: Perhaps it depends on the screen resolution/size but I think you have removed to many things in the "mobile app" block and in the "last topics" block => it looks quite empty in my screen
capture d ecran 2017-11-28 a 21 07 29

I would keep the "mobile app" introduction. In the "last topics" block I think the "date"/"age" is relevant and consistent with the activity feed cards (that do have a "date"/"age"). In my opinion you have enough space to have the thead title on a first line and the last author + age on a second line (could use a smaller fontsize).

File: less/variables.less [hunk 4]
```diff
@@ -13,7 +13,7 @@
 @C2C-orange: #FFAA45;
 @bg-color1 : #FBFBFB;
 @page-header-height: 48px;
-@menu-width: 250px;
+@menu-width: 190px;
```
Comment (asaunier) [inline less/variables.less]: OK but then please recenter the logo:
capture d ecran 2017-11-28 a 21 14 05

File: less/cards.less (hunk 2 above)
Comment (stef74) [reply less/cards.less]: Done !

File: c2corg_ui/templates/index.html (hunk 3 above)
Comment (stef74) [reply c2corg_ui/templates/index.html]: Ok I propose to readd the age for screen more than 1600px 

For mobile bloc ... it's a complain of some people, I reask to forum

File: less/variables.less (hunk 4 above)
Comment (stef74) [reply less/variables.less]: Done !

**Comment by stef74** [comment]
I have alsso fix : 
- icon activity hidden on route
- "i" on xreport
- date on xreport

**Pull request merged**